examples/cloud-router-2-aws-connection/README.md

```diff
@@ -1,16 +1,5 @@
-# Fabric Cloud Router to AWS Service Profile Connection
```

Comment:
It shouldn't remove the start of the files that were already present. That should remain how it was.